packages/suite/src/components/suite/Preloader/DatabaseUpgradeModal.tsx

```diff
@@ -16,7 +18,7 @@ interface DatabaseUpgradeModalProps {
     variant: 'blocking' | 'blocked';
 }
 
-const DatabaseUpgradeModal = (props: DatabaseUpgradeModalProps) => (
```

Comment:
Not sure if there is a rule for this but I think I would be better to destructure the props for better readability. Wdyt?